### DISCUSSION TITLE
At which rate can we expect docker image release for minor changes ?

### DISCUSSION THREAD

**Discussion by jerwen** [apache/superset#14017] [Q&A / Help]
I was experiencing an isssue with the newly released feature Global Async Queries regarding the impersonation functionality. A fix was recently merged on to master (ca506e93) and I was wandering when we could expected a release on docker hub (even a minor one).

**Comment by nytai** [comment]
That commit tag should exist on docker hub. You can use at your own risk as we haven’t gone through a full release process with QA involved

**Reply by jerwen** [reply]
Is there an ongoing process of release ? Do you know when we could expect a new release ?
Thank you for your answer
